src/transformers/generation_utils.py

```diff
@@ -411,7 +411,7 @@ def generate(
             )  # shape: (batch_size * num_return_sequences * num_beams, cur_len)
 
         if self.config.is_encoder_decoder:
-            # create empty decoder_input_ids
+            # create empty decoder input_ids
```

Comment:
accidental

Reply:
not accidental - there is no var `decoder_input_ids` :)

Reply:
Or do you prefer to make any changes in core files in separate PRs?

Reply:
I made a separate PR to keep this one focused https://github.com/huggingface/transformers/pull/6959